### ISSUE TITLE
About the train result and test result

### ISSUE THREAD

**Issue by Zacino** [RM-Zhang/SCPNet#3]
Hello author, i tried train model myself on GoogleMap.
I found in the output information of the training phase that the mace12 in train was much higher than the mace12 in test.
also, I used the parameters provided in your code to test googlemap on test set, the mace12 was only 4.23, when I used the train set to test, the mac12 was 8.8. is this normal？

**Comment by RM-Zhang** [comment]
Thanks for your interest about our work.
It is normal, and mainly because the training set has many hard samples that do not have sufficient information for registration.

**Comment by Zacino** [comment]
> Thanks for your interest about our work. It is normal, and mainly because the training set has many hard samples that do not have sufficient information for registration.

Thanks for your reply!

**Comment by dddlong** [comment]
你好张博士我有类似的问题，我完全按照初始train代码在ggmap上进行了训练使用，并分别用model_iter_120000.pth(训练获得)，ggmap.pth（您发布的）|Test size  |   mace12   |  mace11   |   mace22  |
111 8.786806162413177 1.6162028366380983 2.2588462571840027
end testing
|Test size  |   mace12   |  mace11   |   mace22  |
111 4.335641328278963 1.1747374072805181 2.485997016365464
end testing

**Comment by RM-Zhang** [comment]
> 你好张博士我有类似的问题，我完全按照初始train代码在ggmap上进行了训练使用，并分别用model_iter_120000.pth(训练获得)，ggmap.pth（您发布的）|Test size | mace12 | mace11 | mace22 | 111 8.786806162413177 1.6162028366380983 2.2588462571840027 end testing |Test size | mace12 | mace11 | mace22 | 111 4.335641328278963 1.1747374072805181 2.485997016365464 end testing

Thanks for your interest about our work. If you are referring to the result where your self-trained model achieves a mace of 8.78 on the test set, this is likely because SCPNet, as an unsupervised method, may suffer from unstable convergence during training. You can try using a different random seed to retrain the model, which might lead to better performance.

Moreover, if you are interested, we recommend checking out our new work SSHNet[1], which provides better stability and performance compared to SCPNet.

[1] SSHNet: Unsupervised Cross-modal Homography Estimation via Problem Reformulation and Split Optimization. CVPR, 2025.
(https://github.com/Junchen-Yu/SSHNet).
